includes/admin/class-pinterest-for-woocommerce-admin.php

```diff
@@ -367,6 +367,7 @@ private function get_component_settings() {
 				'serviceLoginUrl'          => $this->get_service_login_url(),
 				'createBusinessAccountUrl' => $this->get_create_business_account_url(),
 				'switchBusinessAccountUrl' => $this->get_switch_business_account_url(),
+				'billingSettingsUrl'       => $this->get_advertiser_billing_settings_url(),
```

Comment:
I think we don't need this. We already have access to the advertiser ID in the React JS code. Look for `appSettings.tracking_advertiser`.

Reply:
I will leave it as it is here for symmetry with other Urls.